### PR TITLE
Add script for publishing TPC

### PR DESCRIPTION
**Pull request by jluk-codi**

### CODE REVIEW

**Review by kravvcuCL — APPROVED** [review]

File: reposync/tpc_external_publish.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+for branch in master R5.0; do
+    for component in binary source; do
+        REPO=tungstenfabric-tpc-$branch-$component DL_PATH=./comb-$branch ./reposync.sh
+    done
+    rsync -vre "ssh -i ./id_rsa" ./comb-$branch/*.rpm tpc-sync@148.251.5.90:/var/www/html/tpc-sync-$branch
```
Comment (sszwed) [inline reposync/tpc_external_publish.sh]: The same IP address is used 4 times. Make a variable from it.

Comment (kravvcuCL) [reply reposync/tpc_external_publish.sh]: Done.

File: reposync/env.sh.template [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export PUBLIC_REPO_ADDR=1.1.1.1
```
Comment (sszwed) [inline reposync/env.sh.template:1]: is it correct address?

Comment (kravvcuCL) [reply reposync/env.sh.template:1]: Nope, this is just a template file with example configuration. There's an env.sh file with correct, used values on ci-repo. Consult the documentation on TCP repo sync here: https://review.opencontrail.org/#/c/51283/9/source/admin/repos-registries.rst (section TCP sync).

**Review by sszwed — APPROVED** [review]

**Pull request merged**